karaf/roboconf-karaf-dist-dm/karaf-resources/net.roboconf.messaging.http.cfg

```diff
@@ -0,0 +1,25 @@
+###########################################################################
+#
+# 	Copyright 2014 Linagora, Université Joseph Fourier
+#
+# 	Licensed under the Apache License, Version 2.0 (the "License");
+# 	you may not use this file except in compliance with the License.
+# 	You may obtain a copy of the License at
+#
+#     	http://www.apache.org/licenses/LICENSE-2.0
+#
+# 	Unless required by applicable law or agreed to in writing, software
+# 	distributed under the License is distributed on an "AS IS" BASIS,
+# 	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# 	See the License for the specific language governing permissions and
+# 	limitations under the License.
+###########################################################################
+
+######################################
+# 	The http messaging parameters
+######################################
+
+# The IP address (or hostname) and port of http server.
+net.roboconf.messaging.http.server.ip = localhost
```

Comment:
Nope.
You should not configure the IP and the port. These are defined by the Jetty configuration in Karaf.